pkg/operator/v411_00_assets/bindata.go

```diff
@@ -688,27 +689,15 @@ metadata:
     revision: "REVISION"
 spec:
   initContainers:
-  - name: wait-for-host-port
-    terminationMessagePolicy: FallbackToLogsOnError
-    image: ${IMAGE}
-    imagePullPolicy: IfNotPresent
-    command: ['/usr/bin/timeout', '30', "/bin/bash", "-c"]
-    args:
-    - |
-      echo -n "Waiting for port :10257 to be released."
-      while [ -n "$(lsof -ni :10257)" ]; do
-        echo -n "."
-        sleep 1
-      done
-  - name: wait-for-cpc-host-port
+  - name: wait-for-host-ports
     terminationMessagePolicy: FallbackToLogsOnError
     image: ${IMAGE}
     imagePullPolicy: IfNotPresent
-    command: ['/usr/bin/timeout', '30', "/bin/bash", "-c"]
+    command: ['/usr/bin/timeout', '105', '/bin/bash', '-ec'] # a bit more than 60s for graceful termination + 35s for minimum-termination-duration, 5s extra cri-o's graceful termination period
```

Comment:
graceful termination applies to apiserver, I think we should set 60 s for TIME_WAIT + 5 s for crio